simple/entities/README.md

```diff
@@ -0,0 +1,25 @@
+# Simple Entities Importer
```

Comment:
Add some more user-oriented documentation?  Like what is the context of this tool?